src/core/QCheck.ml

```diff
@@ -154,11 +154,11 @@ module Gen = struct
         lor ((RS.bits st land 3) lsl 60)  (* Top 2 bits *)  (* top bit = 0 *)
 
   let int st = if RS.bool st then - (pint st) - 1 else pint st
-  let int_bound n =
+  let int_bound n st =
     if n < 0 then invalid_arg "Gen.int_bound";
```

Comment:
is there a reason to not check that anymore before `st` is passed?

Reply:
Nope, unintended omission. Will fix and force push.